src/test/java/teammates/test/cases/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -55,7 +78,7 @@ private void testEditResponse() throws Exception {
                                                   "student1InIESFPTCourse@gmail.tmt",
                                                   "student1InIESFPTCourse@gmail.tmt");
 
-        assertEquals("Student 1 self feedback.", fr.getResponseDetails().getAnswerString());
+        assertEquals("<p>Student 1 self feedback.</p>", fr.getResponseDetails().getAnswerString());
```

Comment:
Why the tag is added?

Reply:
It was giving assertionError so I changed it. I don't know how <p> got added.